actions/mysql_query_show_action_test.go

```diff
@@ -47,13 +47,13 @@ func TestMySQLQueryShow(t *testing.T) {
 		b, err := a.Run(ctx)
 		require.NoError(t, err)
 		assert.LessOrEqual(t, 15000, len(b))
-		assert.LessOrEqual(t, len(b), 26800)
+		assert.LessOrEqual(t, len(b), 27000)
 
 		data, err := agentpb.UnmarshalActionQueryResult(b)
 		require.NoError(t, err)
 		t.Log(spew.Sdump(data))
 		assert.LessOrEqual(t, 400, len(data))
-		assert.LessOrEqual(t, len(data), 700)
+		assert.LessOrEqual(t, len(data), 750)
```

Comment:
Hm, I wonder why this values changed?

Reply:
In PR that you mentioned all tests are green and there is no such changes it tests.

Reply:
`make test-race` failed on the PR with this https://github.com/percona/pmm-agent/runs/5464274760?check_suite_focus=true

I should take some time to try and investigate further anyway.